core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -714,6 +720,12 @@ object KafkaConfig {
   val ShareGroupAssignorsDoc = "The server-side assignors as a list of full class names. The first one in the list is considered as the default assignor to be used in the case where the share group does not specify an assignor."
   val ShareGroupPersisterClassNameDoc = "The class name of share persister for share group. The class should implement " +
     "the <code>org.apache.kafka.server.group.share.Persister</code> interface."
+  val ShareCoordinatorStateTopicPartitionsDoc = "Number of partitions for the internal share-group state topic."
```

Comment:
`"The number of partitions for the share-group state topic (should not change after deployment)."`